scrapyd_k8s/launcher/docker.py

```diff
@@ -55,36 +60,44 @@ def schedule(self, project, version, spider, job_id, settings, args):
         )
 
     def cancel(self, project_id, job_id, signal):
-        c = self._get_container(project_id, job_id)
-        if not c:
+        container = self._get_container(project_id, job_id)
+        if not container:
             return None
 
-        prevstate = self._docker_to_scrapyd_status(c.status)
-        if c.status == 'created' or c.status == 'scheduled':
-            c.remove()
-        elif c.status == 'running':
-            c.kill(signal='SIG' + signal)
+        prevstate = self._docker_to_scrapyd_status(container.status)
+        if container.status == 'created' or container.status == 'scheduled':
+            container.remove()
+        elif container.status == 'running':
+            container.kill(signal='SIG' + signal)
         return prevstate
 
-    def _parse_job(self, c):
-        return {
-            'id': c.labels.get(self.LABEL_JOB_ID),
-            'state': self._docker_to_scrapyd_status(c.status),
-            'project': c.labels.get(self.LABEL_PROJECT),
-            'spider': c.labels.get(self.LABEL_SPIDER)
+    def _parse_job(self, container: Container):
+        state = self._docker_to_scrapyd_status(container.status)
+        job = {
+            'id': container.labels.get(self.LABEL_JOB_ID),
+            'state': state,
+            'project': container.labels.get(self.LABEL_PROJECT),
+            'spider': container.labels.get(self.LABEL_SPIDER),
         }
```

Comment:
Why moving this out of the dict?

Reply:
We move the `state` variable out of the dict so that we can use the variable later on to make an evaluation (for adding start/end time). If that is what you were referring to.

Reply:
Ah, sorry, I missed that, thanks for explaining!